gcp_variant_transforms/transforms/variant_to_bigquery.py

```diff
@@ -41,39 +43,38 @@ def process(self, record):
 class VariantToBigQuery(beam.PTransform):
   """Writes PCollection of ``Variant`` records to BigQuery."""
```

Comment:
Please also change this to be ProcessedVariant.

Reply:
Done.